aspnetcore/tutorials/min-web-api.md

```diff
@@ -875,15 +875,63 @@ The following image shows the Postman update:
 
 The sample app implements a single DELETE endpoint using `MapDelete`:
 
-[!code-csharp[](min-web-api/samples/6.x/todo/Program.cs?name=snippet_delete)]
+[!code-csharp[](min-web-api/samples/7.x/todo/Program.cs?name=snippet_delete)]
 
 Use Postman to delete a to-do item:
 
 * Set the method to `DELETE`.
 * Set the URI of the object to delete (for example `https://localhost:5001/todoitems/1`).
 * Select **Send**.
 
-<a name="over-post-v6"></a>
+## Use the MapGroup API
+
+The sample app code repeats the `todoitems` URL node each time it sets up an endpoint. Web APIs often have groups of endpoints with a common URL node, and the <xref:Microsoft.AspNetCore.Builder.EndpointRouteBuilderExtensions.MapGroup%2A> method is available to help organize such groups. It reduces repetitive code and allows for customizing entire groups of endpoints with a single call to methods like <xref:Microsoft.AspNetCore.Builder.AuthorizationEndpointConventionBuilderExtensions.RequireAuthorization%2A> and <xref:Microsoft.AspNetCore.Builder.RoutingEndpointConventionBuilderExtensions.WithMetadata%2A>.
+
+Replace the code in Program.cs with the following code:
+
+:::code language="csharp" source="min-web-api/samples/7.x/todoGroup/Program.cs" id="snippet_all":::
+
+Notice that a `MapGroup()` call sets up the group, and the endpoints omit the `todoitems` prefix:
+
+:::code language="csharp" source="min-web-api/samples/7.x/todoGroup/Program.cs" id="snippet_group":::
+
+Test the endpoints to verify that they work the same.
+
+## Use the TypedResults API
+
+The `Map<HttpVerb>` methods can call route handler methods instead of using lambdas. To see an example, replace the code in *Program.cs* with the following code:
```

Comment:
```suggestion
The `Map<HttpVerb>` methods can call route handler methods instead of using lambdas. To see an example, update *Program.cs* with the following code:
```
or something less tedious than code repeated twice
replace the contents of program.c ...